### PR TITLE
Modify config to support state on disk by default

### LINKED ISSUE #8505: Release 0.42, node stuck with CHECKING for more than 20 minutes, could not reconnect
### Description

Slack report URL

https://swirldslabs.slack.com/archives/C05R1D74CMC/p1694451139476069

Node003 log

http://35.247.76.217:8095/swirlds-automation/release/0.42/4N_2C/UpdateReconnect/20230911-111352-GCP-Daily-Update-Reconnect-4N-2C/Crypto-Update-NI-DisDuringUpdate-RecAftPrepare-1.5k-25m/node0003/swirlds.log

From 15:53 to 16:13, node3 could not become ACTIVE or reconnect with other nodes
```
2023-09-11 15:53:32.623 23050    INFO  PLATFORM_STATUS  <<platform: status-state-machine>> PlatformStatusStateMachine: Platform spent 4.3 m in ACTIVE. Now in CHECKING {"oldStatus":"ACTIVE","newStatus":"CHECKING"} [com.swirlds.logging.payloads.PlatformStatusPayload]
2023-09-11 16:13:31.868 53026    ERROR EXCEPTION        <<platform: signed-state-sentinel>> SignedStateSentinel: old signed state detected, memory leak probable.
null
2023-09-11 16:16:49.108 57952    INFO  RECONNECT        <<platform-core: SyncProtocolWith2 3 to 2>> ReconnectController: Starting ReconnectController
2023-09-11 16:16:49.109 57953    INFO  PLATFORM_STATUS  <<platform: status-state-machine>> PlatformStatusStateMachine: Platform spent 23.3 m in CHECKING. Now in BEHIND {"oldStatus":"CHECKING","newStatus":"BEHIND"} [com.swirlds.logging.payloads.PlatformStatusPayload]
2023-09-11 16:16:49.112 57954    WARN  STARTUP          <<reconnect: reconnect-controller>> ReconnectLearnerThrottle: Node has fallen behind, reconnect is disabled outside of time window, will die {"nodeId":3} [com.swirlds.logging.payloads.UnableToReconnectPayload]
2023-09-11 16:16:49.114 57955    INFO  STARTUP          <<reconnect: reconnect-controller>> SystemExitUtils: System exit requested (BEHIND_RECONNECT_DISABLED)
thread requesting exit: <reconnect: reconnect-controller>
com.swirlds.common.system.SystemExitUtils.exitSystem(SystemExitUtils.java:47)
	at com.swirlds.common.system.SystemExitUtils.exitSystem(SystemExitUtils.java:83)
	at com.swirlds.platform.reconnect.ReconnectLearnerThrottle.exitIfReconnectIsDisabled(ReconnectLearnerThrottle.java:114)
	at com.swirlds.platform.reconnect.ReconnectHelper.prepareForReconnect(ReconnectHelper.java:91)
	at com.swirlds.platform.reconnect.ReconnectController.executeReconnect(ReconnectController.java:102)
	at com.swirlds.platform.reconnect.ReconnectController.run(ReconnectController.java:89)
```

### Steps to reproduce

Run release 0.42 nightly test panel `GCP-Daily-Update-Reconnect-4N-2C`

### Additional context

_No response_

### Hedera network

other

### Version

release 42 test

### Operating system

None

### PR DESCRIPTION
**Pull request by mhess-swl**
~~Closes 8505~~

EDIT: the `records.useConsolidatedFcq` property is causing a test to fail, so we'll get a fix up for that and put it in a separate PR (see issue #8532)

### CODE REVIEW

**Review by iwsimon — APPROVED** [review]
LGTM

**Review by netopyr — APPROVED** [review]
LGTM

**Review by jeromy-cannon — APPROVED** [review]

**Review by qnswirlds — APPROVED** [review]

File: hedera-node/hedera-mono-service/src/test/java/com/hedera/node/app/service/mono/context/properties/BootstrapPropertiesTest.java [hunk 1]
```diff
@@ -522,17 +522,17 @@ class BootstrapPropertiesTest {
             entry(TOKENS_NFTS_MAX_BATCH_SIZE_MINT, 10),
             entry(TOKENS_NFTS_MAX_BATCH_SIZE_BURN, 10),
             entry(TOKENS_NFTS_MAX_METADATA_BYTES, 100),
-            entry(TOKENS_NFTS_MAX_ALLOWED_MINTS, 5000000L),
+            entry(TOKENS_NFTS_MAX_ALLOWED_MINTS, 10_000_000L),
             entry(TOKENS_NFTS_MINT_THORTTLE_SCALE_FACTOR, ScaleFactor.from("5:2")),
-            entry(TOKENS_NFTS_USE_VIRTUAL_MERKLE, false),
+            entry(TOKENS_NFTS_USE_VIRTUAL_MERKLE, true),
```
Comment (poulok) [inline hedera-node/hedera-mono-service/src/test/java/com/hedera/node/app/service/mono/context/properties/BootstrapPropertiesTest.java:527]: Is there an equivalent for fungible tokens?

Comment (mhess-swl) [reply hedera-node/hedera-mono-service/src/test/java/com/hedera/node/app/service/mono/context/properties/BootstrapPropertiesTest.java:527]: @poulok no, we don't have a flag for fungible token

**Pull request merged**